content/en/tracing/custom_instrumentation/cpp.md

```diff
@@ -144,21 +144,6 @@ The value of the environment variable is a comma (or space) separated list of he
 
 If multiple extraction styles are enabled extraction attempt is done on the order those styles are configured and first successful extracted value is used.
 
-### Resource filtering
```

Comment:
Removing all these sections from these pages, let's make sure we add in a link to the eventual Security and Fine-tuning page